src/elements/account/password/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { useEffect } from 'react'
```

Comment:
In `elements` everything are pages following the defined routes (something I'm not very fond of, but at least it's standardized)... and I preferred having `account/password` so it was accessible (rather than defining different routes for each, or having `account/password` returning a 404).